Userland/Games/GameOfLife/main.cpp

```diff
@@ -115,6 +123,13 @@ int main(int argc, char** argv)
         GUI::Application::the()->quit();
     }));
 
+    auto& pattern_menu = menubar->add_menu("&Pattern");
+    patterns_widget.for_each_pattern([&](auto& pattern) {
+        if (pattern.action())
+            pattern_menu.add_action(*pattern.action());
+        return IterationDecision::Continue;
```

Comment:
The return value of this callback is not used in `for_each_pattern()`, so no return is needed